src/widget/wpushbutton.cpp

```diff
@@ -290,9 +290,7 @@ void WPushButton::onConnectedControlChanged(double dParameter, double dValue) {
     Q_UNUSED(dParameter);
     // Enums are not currently represented using parameter space so it doesn't
     // make sense to use the parameter here yet.
-    if (m_iNoStates == 1) {
```

Comment:
This code is only a workaround for one state buttons where the pressed state matches the value.